src/invidious/helpers/errors.cr

```diff
@@ -86,8 +82,13 @@ def error_template_helper(env : HTTP::Server::Context, status_code : Int32, mess
 
   locale = env.get("preferences").as(Preferences).locale
 
-  error_message = translate(locale, message)
-  next_steps = error_redirect_helper(env)
+  error_message = <<-END_HTML
+    <div class="error_message">
+      <h2>#{translate(locale, "error_processing_data_youtube")}</h2>
+      <p>#{translate(locale, message)}</p>
+      #{error_redirect_helper(env)}
+    </div>
+  END_HTML
```

Comment:
Your changes here will show the `Error while processing the data sent by YouTube` message on any `InfoException` while also removing the next steps message on all `InfoException`s

And `InfoException`s are raised in places unrelated to YouTube as well such as when a user inserts the wrong password. 

Related: #4651